### PR TITLE
[issue #12081]  module api upgrade  junit4 to junit5

### LINKED ISSUE #12081: [test] module api upgrade to junit5
This is sub issue of https://github.com/alibaba/nacos/issues/12016 ,module api upgrade to junit5

### PR DESCRIPTION
**Pull request by shalk**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

fix #12081

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: api/src/test/java/com/alibaba/nacos/api/ability/ClientAbilitiesTest.java [hunk 1]
```diff
@@ -20,34 +20,34 @@
...
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class ClientAbilitiesTest {
+class ClientAbilitiesTest {
     
     private static ObjectMapper mapper;
-
```
Comment (KomachiSion) [inline api/src/test/java/com/alibaba/nacos/api/ability/ClientAbilitiesTest.java:32]: dont change the indent.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: api/src/test/java/com/alibaba/nacos/api/ability/ClientAbilitiesTest.java (hunk 1 above)
Comment (shalk) [reply api/src/test/java/com/alibaba/nacos/api/ability/ClientAbilitiesTest.java:32]: done

**Review by KomachiSion — DISMISSED** [review]

**Comment by KomachiSion** [comment]
There is one class conflict, simple fix it will be ready to merged.

**Comment by shalk** [comment]
It is strange for IT test fail only for JDK8 latest. But JDK8.0.192 is pass.
It maybe effect by  5169f06654a57db4f01cde83406bbc1ba07d4a29

**Review by KomachiSion — APPROVED** [review]

**Comment by KomachiSion** [comment]
> It is strange for IT test fail only for JDK8 latest. But JDK8.0.192 is pass. It maybe effect by [5169f06](https://github.com/alibaba/nacos/commit/5169f06654a57db4f01cde83406bbc1ba07d4a29)

TLS is depend os and jdk version. If jdk has some change for tls support or has some bug in jdk. It might effect test and runtime.

**Pull request merged**